jsapp/js/components/formViewSideTabs.es6

```diff
@@ -162,6 +162,20 @@ class FormViewSideTabs extends Reflux.Component {
           path: ROUTES.FORM_REST.replace(':uid', this.state.asset.uid),
         });
       }
+
+      // TODO: Fix permissions
```

Comment:
> I noticed we'll need to setup permissions for the route elements and that's probably gonna take me some time, so I've created a [new task](https://www.notion.so/kobotoolbox/Setup-history-logs-permissions-fe-be-1127e515f65480a4b51af7ddc0765a13?pvs=4) for that.

I think you can just ask John/Jackie/Tino about what permissions should be applied here. `userCan` is a bit complex inside, but the usage is pretty straightforward. Just swap these with whatever they say we need here. Also take a look at `/jsapp/js/components/permissions/permConstants.ts` - there is a list of all permissions, each with a comment describing in some detail what is it about :)

Reply:
But it's ok to have this merged with the TODO comments - if it would make your work easier - as you will wait for the response :)